diff -Naur protobuf-3.5.0/src/google/protobuf/stubs/atomicops_internals_generic_c11_atomic.h protobuf-3.5.0_patched/src/google/protobuf/stubs/atomicops_internals_generic_c11_atomic.h
--- protobuf-3.5.0/src/google/protobuf/stubs/atomicops_internals_generic_c11_atomic.h	2017-11-13 19:47:29.000000000 +0100
+++ protobuf-3.5.0_patched/src/google/protobuf/stubs/atomicops_internals_generic_c11_atomic.h	2025-07-30 14:15:59.956298045 +0200
@@ -56,7 +56,7 @@
 #if defined(__GLIBCXX__)
   // Work around libstdc++ bug 51038 where atomic_thread_fence was declared but
   // not defined, leading to the linker complaining about undefined references.
-  __atomic_thread_fence(std::memory_order_seq_cst);
+  __atomic_thread_fence(__ATOMIC_SEQ_CST);
 #else
   std::atomic_thread_fence(std::memory_order_seq_cst);
 #endif
